vllm_spyre/envs.py

```diff
@@ -0,0 +1,61 @@
+import os
+from typing import TYPE_CHECKING, Any, Callable, Dict
+
+if TYPE_CHECKING:
+    VLLM_SPYRE_DYNAMO_BACKEND: str = "sendnn_decoder"
+    # TODO uncomment those when scheduler is ready
+    # VLLM_SPYRE_WARMUP_PROMPT_LENS: Optional[List[int]] = None
+    # VLLM_SPYRE_WARMUP_NEW_TOKENS: Optional[List[int]] = None
+    # VLLM_SPYRE_WARMUP_BATCH_SIZES: Optional[List[int]] = None
+
```

Comment:
We should avoid having large commented-out blocks of code on main. If there is a desire to have this prepared for later when the scheduler plugin is ready, we can keep it on a branch somewhere.